src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -2113,6 +2159,35 @@ components:
             typed:
               $ref: "#/components/schemas/MetadataTypedValue"
 
+    AccountLockerResourceVaultCollection:
+      description: TBD
+      allOf:
+        - $ref: "#/components/schemas/ResultSetCursorMixin"
```

Comment:
I assume we will be able to provide sensible values here if a user specifies `resource_addresses`?

Reply:
Yeah, should the client specify `explicit_resource_addresses` we'll return responses for those that were requested.

Reply:
Sorry my comment wasn't very clear. I meant to the `ResultSetCursorMixin` - e.g. no cursor, size, or whatever it includes?

Reply:
Yeah, it should work just fine. However please note that in theory we could expect configure max_page_size = 10 and max_explicit_resource_address = 50 and then you'd have to use cursor to iterate over 5 pages of explicitly requested resources. :)